Makefile

```diff
@@ -60,6 +60,6 @@ setup:
 test:
 	$(ISTANBUL) cover node_modules/.bin/_mocha -- --recursive --timeout 10000
 	$(ISTANBUL) check-coverage --branches 100
-ifneq ($(shell node --version | sed 's/[.][^.]*$$//'),v0.10)
+ifeq ($(shell node --version | cut -d . -f 1),v6)
```

Comment:
My primary environment (currently) doesn't actually support *any* escape codes, but I still think they're a good idea here. (If you were very worried about this, you could have a `$TERM=dumb` check)